uktrade/data-hub-frontend
# feature/cls2-431-project-task-test-refactoring

## Description of change

- Move some of the functional tests for the form validation into a component test.
- Move the investment project task form into separate component, ready for the edit task page

## Test instructions

You should be able to add and view investment project tasks in the same way as the dev site

## Screenshots
NA

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
